src/Command/Checker.php

```diff
@@ -94,8 +96,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('<bg=green;fg=white>    Everything is OK!     </>');
     }
 
-    private function docIsCompatible($headerValidator, $fileContent)
+    private function docIsCompatible($headerValidator, $fileContent, $docheaderFile)
     {
-        return $headerValidator->__invoke($fileContent) || false !== strpos($fileContent, $this->header);
+        return $headerValidator->__invoke($fileContent) || false !== strpos($fileContent, $docheaderFile);
+    }
+
+    private function getDocheaderFileContent(InputInterface $input)
+    {
+        $docheaderFile = $input->getOption('docheader');
+
+        if ('.docheader' === $docheaderFile) {
+            $docheaderFile = getcwd() . '/' . $docheaderFile;
```

Comment:
You may want to use `DIRECTORY_SEPARATOR` instead to ensure this works on Windows and other non-POSIX systems.